### PR TITLE
Automator: add job to update istio.io ref docs nightly

### LINKED ISSUE #1626: Create a periodic job to update istio.io reference docs
We need a nightly job that updates istio.io's reference docs. To do this, the job needs to check out the istio/istio.io repo and run the scripts/grab_reference_docs.sh script. If after running the script any files are modified in the repo, create a PR for the changes.

This only needs to work on the master branch.

### PR DESCRIPTION
**Pull request by clarketm**
Dependent on changes to automator in #2223
/hold

fix #1626

### CODE REVIEW

File: prow/cluster/jobs/all-periodics.yaml [hunk 1]
```diff
@@ -326,3 +326,48 @@ periodics:
...
+      - --token-path=/etc/github-token/oauth
+      - --cmd=make update_ref_docs
+      image: gcr.io/istio-testing/build-tools:master-2019-12-09T23-37-05
+      resources:
+        limits:
+          cpu: "3"
+          memory: 24Gi
+        requests:
+          cpu: 500m
+          memory: 3Gi
```
Comment (scottilee) [inline prow/cluster/jobs/all-periodics.yaml]: Just out of curiosity, why do you provide so much memory?

**Comment by clarketm** [comment]
@geeknoid 
Can you please verify this job satisfies the requirements of the original issue you created (i.e. #1626)

cc @istio/wg-test-and-release-maintainers PTAL

**Comment by clarketm** [comment]
@istio/wg-test-and-release-maintainers
Can I please get a review.

**Review by howardjohn — APPROVED** [review]

**Pull request merged**